setup.cfg

```diff
@@ -51,6 +59,17 @@ console_scripts =
 [flake8]
 max-line-length = 88
 
+[tool:pytest]
+# Run pytest with all our checkers, and don't spam us with massive tracebacks on error
+addopts =
+    --tb=native -vv --flake8 --black --mypy --doctest-modules --doctest-glob="*.rst"
+     --cov=pmacfiltercontrol --cov-report term --cov-report xml:cov.xml
+asyncio_mode = strict
+
+[coverage:run]
+# This is covered in the versiongit test suite so exclude it here
+omit = */_version_git.py
```

Comment:
We are using `setuptools_scm` not `version_git`, so this should be `_version.py`